### ISSUE TITLE
Loading previously made topic-model

### ISSUE THREAD

**Issue by Akhamis01** [MaartenGr/BERTopic#348]
Hello,

Whenever I create a topic-model, it seems that the topics constantly change even when the docs stay the same (roughly 20-30k docs). To my knowledge, I could save the model using `model.save('my_model')` then load it later using `BERT.load('my_model')`.

However, this method takes up quite a lot of storage as I need to do this for many large models, so I was wondering if there is a more efficient way of doing this? For example, the [LDA](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.LatentDirichletAllocation.html) library has a `random_state ` parameter which would reproduce the same model if the same seed is passed in. I was wondering if BERTopic has a parameter or a feature similar to this?

Thank you!

**Comment by MaartenGr** [comment]
Indeed, the results are not consistent between runs. This is because BERTopic uses UMAP which is stochastic by nature. You can find a bit more about that [here](https://maartengr.github.io/BERTopic/faq.html#why-are-the-results-not-consistent-between-runs). If you would like to use a `random_state`, then you will have to do this within UMAP and pass it to BERTopic. Doing this though comes with a number of side-effects that you can read about [here](https://umap-learn.readthedocs.io/en/latest/reproducibility.html). 

When saving the model, you can set `save_embedding_model=False` in order to prevent saving the embedding model. Typically, this is not necessary as it can download the model on the fly or when you have the embedding model stored locally.

**Comment by Akhamis01** [comment]
Thank you very much!
